include/swift/Sema/ConstraintLocator.h

```diff
@@ -868,6 +868,18 @@ class ConstraintLocatorBuilder {
     return (getSummaryFlags() & ConstraintLocator::IsNonEphemeralParam);
   }
 
+  /// Checks whether this locator is describing an anchor that produces a value of a
+  /// callable type that could be implicit called using "()" either by defining
```

Comment:
```suggestion
  /// callable type that could be called using "()" either by defining
```

If one writes `()`, that's just a call, not an implicit call, right?